### PR TITLE
Add EfficientNet encoder

### PR DESCRIPTION
**Pull request by SiarheiFedartsou**
Hi! Added EfficientNet encoder from here https://github.com/lukemelas/EfficientNet-PyTorch/tree/master/efficientnet_pytorch
Let me know if any changes needed.

### CODE REVIEW

File: segmentation_models_pytorch/encoders/efficientnet.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+from efficientnet_pytorch import EfficientNet
+from efficientnet_pytorch.utils import relu_fn, url_map, get_model_params
+import torch.nn as nn
+import torch
+import segmentation_models_pytorch as smp
```
Comment (qubvel) [inline segmentation_models_pytorch/encoders/efficientnet.py]: Why do we need this import?

File: segmentation_models_pytorch/encoders/efficientnet.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        super().load_state_dict(state_dict, **kwargs)
+
+
+
+def _get_pretrained_settings(encoder):
+    pretrained_settings = {
+        'imagenet': {
+            'mean': [0.485, 0.456, 0.406],
+            'std': [0.229, 0.224, 0.225],
+            'url': url_map[encoder]
```
Comment (qubvel) [inline segmentation_models_pytorch/encoders/efficientnet.py]: Please, add here:
```
'input_space': 'RGB',
'input_range': [0, 1],
```

File: segmentation_models_pytorch/encoders/efficientnet.py (hunk 1 above)
Comment (SiarheiFedartsou) [reply segmentation_models_pytorch/encoders/efficientnet.py]: Just forgot to remove it after porting this code from my "external" implementation :) Fixed.

File: segmentation_models_pytorch/encoders/efficientnet.py (hunk 2 above)
Comment (SiarheiFedartsou) [reply segmentation_models_pytorch/encoders/efficientnet.py]: Done.

**Comment by qubvel** [comment]
Great! Thanks for contribution!

**Pull request merged**